### ISSUE TITLE
Question: Is there a reason that the 'mr create' log is the way it is?

### ISSUE THREAD

**Issue by prarit** [zaquestion/lab#464]
@zaquestion , just curious.  Is there a reason that the comment in 'mr create' looks the way it does? 

For example,

```
# Changes:
#
# 85e7098a84f6 (Prarit Bhargava, 10 minutes ago)
#    Makefile change
# 
# 46f99955f7af (Prarit Bhargava, 27 hours ago)
#    Prarit's USB Change
```

However 'git pull-request' outputs
```

Prarit Bhargava (2):
      Prarit's USB Change
      Makefile change
```

Is there any reason that the log looks the way it does?  Or is it a 'just because'?

I'd like to change it so that it looks more like the output of git-pull request.  At a minimum I think adding a commented diffstat would be good to do.

**Comment by prarit** [comment]
The more I look at this the more I think I'd do a combination of the two.  

I would be good to have

# 85e7098a84f6 (Prarit Bhargava, 10 minutes ago)
#    Makefile change
# 
# 46f99955f7af (Prarit Bhargava, 27 hours ago)
#    Prarit's USB Change
# 
# Makefile                  | 2 +-
# drivers/usb/core/buffer.c | 2 ++
# 2 files changed, 3 insertions(+), 1 deletion(-)

The benefit is that users could uncomment the changelog for inclusion if they wanted to.  I'm not convinced the date needs to be there though.

**Comment by zaquestion** [comment]
`mr create`'s commented changelog used to match what hubs looked like. These days it looks like they've simplified the format. Personally I sorta liked having all the details, but I'm open to changes. Your latest comment on here looks appealing :+1:

**Comment by zaquestion** [comment]
https://github.com/zaquestion/lab/pull/469

**Comment by prarit** [comment]
Resolved in #469
